### ISSUE TITLE
Extra code in a JSON Object (actionCode)

### ISSUE THREAD

**Issue by Nava2** [umple/umple#440]
Originally reported on Google Code with ID 542

---

**What steps will reproduce the problem?**
1. Move a class diagram
2. Find the action: addPositioning
3. See the actionCode sent to the compile.php
4. There are other examples that will cause this behavior.

**What is the expected output? What do you see instead?**
I should expect a clean JSON format object like this: {...}
Instead, I see this: <p>URL_SPLIT{....}

**Please provide any additional information below.**
Basically, I need to remove the <p>URL_SPLIT 
In the server-side, this will always make the ParseResult of the JSON to fail.

---

Reported by @charlesmwang on 2014-04-21 06:15:41

**Comment by Nava2** [comment]
Solved in Issue 401

---

Reported by @charlesmwang on 2014-04-21 08:04:21

**Comment by Nava2** [comment]
The problem was that in the Action.updateUmpleLayoutEditor in umple_action.js , 
it did not parse the result as the result contains <p>URL_SPLIT . 

It directly attached the result.

Resolved it by splitting the code and extracting the json component.

Resolved in Issue 401.

---

Reported by `charleshayashi` on 2014-04-21 16:29:55

**Comment by Nava2** [comment]
Reported by @umple on 2014-05-01 13:47:29
- **Status changed**: `Verified`
